PPL/costAnalisis.md

```diff
@@ -0,0 +1,1724 @@
+   <td><strong>Bruto</strong>
+   </td>
+   <td><strong>Neto</strong>
+   </td>
+   <td><strong>Bruto</strong>
+   </td>
+   <td><strong>Neto</strong>
+   </td>
+   <td><strong>Bruto</strong>
+   </td>
+   <td><strong>Neto</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Project Manager
+   </td>
+   <td>23,50€
+   </td>
+   <td>16,45€
+   </td>
+   <td>235€ 
+   </td>
+   <td>164,50€
+   </td>
+   <td>940€
+   </td>
+   <td>658€
+   </td>
+  </tr>
+  <tr>
+   <td>Team Lead
+   </td>
+   <td>18,25€
+   </td>
+   <td>12,78€
+   </td>
+   <td>182,50€
+   </td>
+   <td>127,80€
+   </td>
+   <td>730€
+   </td>
+   <td>511€
+   </td>
+  </tr>
+  <tr>
+   <td>Junior Developer
+   </td>
+   <td>10,50€
+   </td>
+   <td>7,35€
+   </td>
+   <td>105€
+   </td>
+   <td>73,50€
+   </td>
+   <td>420€
+   </td>
+   <td> 294€
+   </td>
+  </tr>
+  <tr>
+   <td>Community Manager
+   </td>
+   <td>10,26€
+   </td>
+   <td>7,18€
+   </td>
+   <td>102,6€
+   </td>
+   <td>71,80€
+   </td>
+   <td>410,40€
+   </td>
+   <td>287,28€
+   </td>
+  </tr>
+</table>
+
+<br/>
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td colspan="12" ><strong>Costes salariales totales del equipo</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td colspan="6" ><strong>Bruto</strong>
+   </td>
+   <td colspan="6" ><strong>Neto</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Rol</strong>
+   </td>
+   <td><strong>Nº</strong>
+   </td>
+   <td colspan="2" ><strong>/h</strong>
+   </td>
+   <td colspan="2" ><strong>/s</strong>
+   </td>
+   <td colspan="2" ><strong>/m</strong>
+   </td>
+   <td colspan="2" ><strong>/h</strong>
+   </td>
+   <td colspan="2" ><strong>/s</strong>
+   </td>
+   <td colspan="2" ><strong>/m</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Project Manager
+   </td>
+   <td>1
+   </td>
+   <td colspan="2" >23,50€
+   </td>
+   <td colspan="2" >235€
+   </td>
+   <td colspan="2" >940€
+   </td>
+   <td colspan="2" >16,45€
+   </td>
+   <td colspan="2" >164,5€
+   </td>
+   <td colspan="2" >658€
+   </td>
+  </tr>
+  <tr>
+   <td>Team Lead
+   </td>
+   <td>2
+   </td>
+   <td colspan="2" >18,25€
+   </td>
+   <td colspan="2" >182,50€
+   </td>
+   <td colspan="2" >730€
+   </td>
+   <td colspan="2" >12,78€
+   </td>
+   <td colspan="2" >127,80€
+   </td>
+   <td colspan="2" >511€
+   </td>
+  </tr>
+  <tr>
+   <td>Junior Dev
+   </td>
+   <td>13
+   </td>
+   <td colspan="2" >10,50€
+   </td>
+   <td colspan="2" >136,5€ 
+   </td>
+   <td colspan="2" >546€
+   </td>
+   <td colspan="2" >7,35€
+   </td>
+   <td colspan="2" >102,90€
+   </td>
+   <td colspan="2" >411,6€
+   </td>
+  </tr>
+  <tr>
+   <td colspan="2" ><strong>Total</strong>
+   </td>
+   <td colspan="2" >175,5€
+   </td>
+   <td colspan="2" >2.521,50€
+   </td>
+   <td colspan="2" >10.086€
+   </td>
+   <td colspan="2" >137,56€
+   </td>
+   <td colspan="2" >1.757,80€
+   </td>
+   <td colspan="2" >6.619,20€
+   </td>
+  </tr>
+</table>
+
+<br/>
+
+
+<table>
```

Comment:
Though I agree that community manager is outside of the team expenses, please update this table with the salary of the CM

Reply:
done it!